nomenclature/definitions/variable/economy/economy.yaml

```
@@ -4,6 +4,79 @@ Consumption:
    description: Total consumption of all goods, by all consumers in a region
    unit: billion US$2010/yr
 
+Consumption|Households:
+   description: Total consumption of all goods, by households in a region
+   unit: billion US$2010/yr
+
+Consumption|Government:
+   description: Total consumption of all goods, by the government in a region
+   unit: billion US$2010/yr
+
+Consumption|Industry:
+   description: Total consumption of all goods, by industries in a region
+   unit: billion US$2010/yr
+
+Consumption|Households|<Product>|Imported:
+   description: Consumption of imported <this product> by households
+   unit: million euro/yr
+
+Consumption|Government|<Product>|Domestic:
+   description: Consumption of domestic <this product> by households
+   unit: million euro/yr
+
+Consumption|Government|<Product>|Imported:
+   description: Consumption of imported <this product> by government
+   unit: million euro/yr
+
+Consumption|Government|<Product>|Domestic:
+   description: Consumption of domestic <this product> by government
+   unit: million euro/yr
+
+Consumption|Gross Capital Formation|<Product>|Imported:
+   description: Consumption of imported <this product> by gross capital formation
+   unit: million euro/yr
+
+Consumption|Gross Capital Formation|<Product>|Domestic:
+   description: Consumption of domestic <this product> by gross capital formation
+   unit: million euro/yr
+
+Consumption|Industry|<Product>|Imported:
+   description: Consumption of imported <this product> by industries
+   unit: million euro/yr
+
+Consumption|Industry|<Product>|Domestic:
+   description: Consumption of domestic <this product> by industries
+   unit: million euro/yr
+
+Output|Industry|<Industry>:
+   description: Total monetary value of production (output) by <this industry>.
+     The value includes cost of production, capital costs, labor costs and taxes.
+   unit: million euro/yr
+
+Output|Product|<Product>:
+   description: Total monetary value of <this product> produced (output) and sold to industry, households and government
+   unit: million euro/yr
+
+Capital|<Industry>:
+   description: Total capital costs spend by <this industry>
+   unit: million euro/yr
+
+Labor|<Industry>:
+   description: Total wages spend by <this industry>
+   unit: million euro/yr
+
+Product Price Index|<Product>:
+   description: Price index on <this product> level, relative to base year 2011
+   unit: index equals 1 in 2011, the reference year
```

sorry, only realized this now
```suggestion
   note: The index equals 1 in 2011, the reference year
   unit: ''
```